atomic_reactor/schemas/container.json

```diff
@@ -5,6 +5,33 @@
 
   "type": ["object", "null"],
   "properties": {
+    "go": {
```

Comment:
Any reason for not specifying the type for "go"?

Reply:
Oh, missed it. I'll add it.

Reply:
Fixed.